utils/email.js

```diff
@@ -0,0 +1,9 @@
+export function reformatEmail(email) {
+  if (typeof email === "string") {
+    if (email) {
+      return email.replace("@umail.ucsb.edu", "@ucsb.edu");
+    } else {
+      throw "Null string";
+    }
+  }
+}
```

Comment:
Do we want to throw in the case where the email is not a string? It's also better practice to wrap the error message in a `new Error` constructor instead of throwing a raw string.